services/QuillLMS/Gemfile

```diff
@@ -153,6 +153,9 @@ gem 'paperclip'
 gem 'rb-readline'
 gem 'awesome_print'
 
+# TODO: to be removed with Rails 7
```

Comment:
Just to clarify, is there a migration path from this to what Rails 7 uses?

Also, we might want to clarify in this comment that Rails has similar functionality to this gem, so we'd want to migrate our usage to the Rails 7 structure, then remove the gem.

Reply:
For a migration path, my plan was to create new fields on the  `client_id` and `client_secret` and copy the unencrypted values over to those new fields.  Then remove `client_id_cipher_text` and `client_secret_cipher_text`.

Reply:
I've updated the TODO comment.